flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/stream/sql/JoinITCase.scala

```diff
@@ -66,6 +68,29 @@ class JoinITCase(state: StateBackendMode) extends StreamingWithStateTestBase(sta
       .toTable(tEnv, 'b1, 'b2, 'b3, 'b4, 'b5)
     tEnv.registerTable("A", tableA)
     tEnv.registerTable("B", tableB)
+
+    val dataId1 = TestValuesTableFactory.registerData(TestData.data2_1)
+    tEnv.executeSql(s"""
+                       |create table l (
+                       |  a int,
+                       |  b double
+                       |) with (
+                       |  'connector' = 'values',
+                       |  'data-id' = '$dataId1',
+                       |  'bounded' = 'true'
```

Comment:
These tests are all bounded source, but this rule support both stream and batch. Whether the unbounded source database need to add to verify the rule in streaming mode?

Reply:
not necessary, the correctness is not affected by bounded source and unbounded source